persistent-postgresql/persistent-postgresql.cabal

```
@@ -1,5 +1,5 @@
 name:            persistent-postgresql
-version:         2.11.0.0
+version:         2.11.0.1
```

We don't need a version bump here because we haven't released 2.11.0.0 yet.
```suggestion
version:         2.11.0.0
```